pkg/rehearse/jobs.go

```diff
@@ -101,12 +101,19 @@ func makeRehearsalPresubmit(source *prowconfig.Presubmit, repo string, prNumber
 		branch = strings.TrimPrefix(strings.TrimSuffix(source.Branches[0], "$"), "^")
 		if len(repo) > 0 {
 			orgRepo := strings.Split(repo, "/")
-			rehearsal.ExtraRefs = append(rehearsal.ExtraRefs, pjapi.Refs{
-				Org:     orgRepo[0],
-				Repo:    orgRepo[1],
-				BaseRef: branch,
-				WorkDir: true,
-			})
+			jobOrg := orgRepo[0]
+			jobRepo := orgRepo[1]
+
+			if refs != nil {
```

Comment:
I think we need to add `ExtraRefs` when `refs==nil` though, for periodics?

Reply:
there is no org/repo to add. In the periodics, the user has to add the `extra_refs` by himself.

Reply:
Ah, we're relying on `repo==""` for periodics...